vernac/metasyntax.ml

```diff
@@ -1862,15 +1862,11 @@ let warn_custom_entry =
          (fun s ->
           strbrk "Custom entry " ++ str s ++ strbrk " has been overridden.")
 
-let load_custom_entry _ _ = ()
-
-let open_custom_entry _ (_,(local,s)) =
+let load_custom_entry _ (_,(local,s)) =
   if Egramcoq.exists_custom_entry s then warn_custom_entry s
```

Comment:
Isn't this going to trigger a lot of warning in case of diamonds?

Reply:
Not sure what you mean, example?

Reply:
If you require a library defining a custom entry twice you'll get a warning. Before, you had to import it which is harder to do involuntarily, except maybe for `Export` statements. There is the same kind of problem with all superglobal objects btw.

Reply:
I believe redundant requires are skipped, so no warning.

Reply:
Isn't it correct to warn if the entry is defined twice?
If you meant requiring twice AFAIK the second require is skipped.